Hello 2 1
2021-04-18 09:58:19.876645: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2021-04-18 09:58:19.876675: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
WARNING:tensorflow:From /home/parth/.local/lib/python3.8/site-packages/tensorflow/python/compat/v2_compat.py:96: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
Started Reading Flags

checkpoint_load_mode = f_d_U
d_pickle_name = d_processed.p
data_dir = /home/parth/Desktop/SEM6/RnD/Learning-From-Rules/data/TREC
dropout_keep_prob = 1.0
early_stopping_p = 20
f_d_adam_lr = 0.0001
f_d_batch_size = 16
f_d_class_sampling_str = 10,10,10,10,10,10
f_d_epochs = 4
f_d_metrics_pickle_name = metrics_train_f_on_d.p
f_d_primary_metric = accuracy
f_d_U_adam_lr = 0.0001
f_d_U_batch_size = 32
f_d_U_epochs = 4
f_d_U_metrics_pickle_name = metrics_train_f_on_d_U.p
f_infer_out_pickle_name = infer_f.p
f_layers_str = 512,512
gamma = 0.1
lamda = 0.01
min_rule_coverage = 0
network_dropout = True
num_checkpoints = 1
num_classes = 0
num_load_d = 0
num_load_U = 0
num_load_validation = 0
output_dir = /home/parth/Desktop/SEM6/RnD/Learning-From-Rules/data/TREC/outputs/learn2reweight_0.1_0.01_1
rule_classes_str = 
run_mode = test_f
shuffle_batches = True
test_w_batch_size = 1000
U_pickle_name = U_processed.p
use_joint_f_w = False
validation_pickle_name = test_processed.p
w_infer_out_pickle_name = infer_w.p
w_layers_str = 512,512

Hi1
Ended Reading Flags
Loading from hoff  /home/parth/Desktop/SEM6/RnD/Learning-From-Rules/data/TREC/outputs/learn2reweight_0.1_0.01_1/d_processed.p
Loading from hoff  /home/parth/Desktop/SEM6/RnD/Learning-From-Rules/data/TREC/U_processed.p
Rule classes:  [2, 1, 1, 4, 3, 2, 1, 3, 5, 4, 3, 2, 4, 1, 2, 5, 4, 1, 2, 4, 3, 1, 3, 2, 1, 1, 4, 4, 5, 1, 3, 5, 2, 1, 5, 1, 5, 5, 0, 5, 5, 1, 4, 2, 3, 2, 1, 3, 5, 1, 1, 3, 2, 1, 1, 5, 2, 3, 2, 5, 3, 3, 4, 5, 5, 4, 2, 0]
length of covered U: 4637
Size of d before oversampling:  68
Size of U (covered) :  4637
Size of d after oversampling:  680
Size of d_U after combining:  5317
Loading 68 elements from d
num instances in d:  680
Loading from hoff  /home/parth/Desktop/SEM6/RnD/Learning-From-Rules/data/TREC/test_processed.p
Setting value of d to 0 for test data
test_w len:  481
test_w len:  481
Number of features:  1024
Number of classes:  6
Print num of rules to train:  68
Print num of rules:  68



[Errno 2] No such file or directory: '/home/parth/Desktop/SEM6/RnD/Learning-From-Rules/data/TREC/outputs/learn2reweight_0.1_0.01_1/metrics_train_f_on_d.p'
Did not find prev best metric for run type f_d. Setting to zero.
Found prev best metric for run type f_d_U: 0.524
best metrics dict:  {'f1_score_1': 0.5126353790613719, 'precision_1': 0.461038961038961, 'recall_1': 0.5772357723577236, 'support_1': 123, 'f1_score': array([0.        , 0.51263538, 0.33160622, 0.61797753, 0.64150943,
       0.57297297]), 'precision': array([0.        , 0.46103896, 0.39506173, 0.80882353, 0.6       ,
       0.47321429]), 'recall': array([0.        , 0.57723577, 0.28571429, 0.5       , 0.68918919,
       0.7260274 ]), 'support': array([  8, 123, 112, 110,  74,  73]), 'avg_f1_score': 0.4461169219505128, 'avg_precision': 0.4563564174266788, 'avg_recall': 0.46302777408691204, 'accuracy': 0.524, 'epoch': 3.0, 'f_d_global_step': 498}
/home/parth/.local/lib/python3.8/site-packages/tensorflow/python/keras/legacy_tf_layers/core.py:171: UserWarning: `tf.layers.dense` is deprecated and will be removed in a future version. Please use `tf.keras.layers.Dense` instead.
  warnings.warn('`tf.layers.dense` is deprecated and '
/home/parth/.local/lib/python3.8/site-packages/tensorflow/python/keras/engine/base_layer_v1.py:1719: UserWarning: `layer.apply` is deprecated and will be removed in a future version. Please use `layer.__call__` method instead.
  warnings.warn('`layer.apply` is deprecated and '
WARNING:tensorflow:From /home/parth/.local/lib/python3.8/site-packages/tensorflow/python/util/dispatch.py:201: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
WARNING:tensorflow:From /home/parth/.local/lib/python3.8/site-packages/tensorflow/python/util/dispatch.py:201: softmax_cross_entropy_with_logits (from tensorflow.python.ops.nn_ops) is deprecated and will be removed in a future version.
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See `tf.nn.softmax_cross_entropy_with_logits_v2`.

WARNING:tensorflow:From /home/parth/Desktop/SEM6/RnD/Data_Pgm_Subset_Slc/Implyloss/my_model.py:615: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.
[
    "<tf.Variable 'global_step:0' shape=() dtype=int64_ref>",
    "<tf.Variable 'f_d_global_step:0' shape=() dtype=int32_ref>",
    "<tf.Variable 'f_network/dense/kernel:0' shape=(1024, 512) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense/bias:0' shape=(512,) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_1/kernel:0' shape=(512, 512) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_1/bias:0' shape=(512,) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_2/kernel:0' shape=(512, 6) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_2/bias:0' shape=(6,) dtype=float32_ref>",
    "<tf.Variable 'beta1_power:0' shape=() dtype=float32_ref>",
    "<tf.Variable 'beta2_power:0' shape=() dtype=float32_ref>",
    "<tf.Variable 'f_network/dense/kernel/f_d_Adam:0' shape=(1024, 512) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense/kernel/f_d_Adam_1:0' shape=(1024, 512) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense/bias/f_d_Adam:0' shape=(512,) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense/bias/f_d_Adam_1:0' shape=(512,) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_1/kernel/f_d_Adam:0' shape=(512, 512) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_1/kernel/f_d_Adam_1:0' shape=(512, 512) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_1/bias/f_d_Adam:0' shape=(512,) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_1/bias/f_d_Adam_1:0' shape=(512,) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_2/kernel/f_d_Adam:0' shape=(512, 6) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_2/kernel/f_d_Adam_1:0' shape=(512, 6) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_2/bias/f_d_Adam:0' shape=(6,) dtype=float32_ref>",
    "<tf.Variable 'f_network/dense_2/bias/f_d_Adam_1:0' shape=(6,) dtype=float32_ref>",
    "<tf.Variable 'f_d_U_global_step:0' shape=() dtype=int32_ref>",
    "<tf.Variable 'w_network/w_layer_0/kernel:0' shape=(1092, 512) dtype=float32_ref>",
    "<tf.Variable 'w_network/w_layer_0/bias:0' shape=(512,) dtype=float32_ref>",
    "<tf.Variable 'w_network/w_layer_1/kernel:0' shape=(512, 512) dtype=float32_ref>",
    "<tf.Variable 'w_network/w_layer_1/bias:0' shape=(512,) dtype=float32_ref>",
    "<tf.Variable 'w_network/w_linear_layer/kernel:0' shape=(512, 1) dtype=float32_ref>",
    "<tf.Variable 'w_network/w_linear_layer/bias:0' shape=(1,) dtype=float32_ref>"
]
2021-04-18 09:58:22.117338: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-04-18 09:58:22.117750: I tensorflow/compiler/jit/xla_gpu_device.cc:99] Not creating XLA devices, tf_xla_enable_xla_devices not set
2021-04-18 09:58:22.117873: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory
2021-04-18 09:58:22.117894: W tensorflow/stream_executor/cuda/cuda_driver.cc:326] failed call to cuInit: UNKNOWN ERROR (303)
2021-04-18 09:58:22.117921: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (parth-G3-3590): /proc/driver/nvidia/version does not exist
2021-04-18 09:58:22.206945: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:196] None of the MLIR optimization passes are enabled (registered 0 passes)
2021-04-18 09:58:22.212121: I tensorflow/core/platform/profile_utils/cpu_utils.cc:112] CPU Frequency: 2599990000 Hz
Restoring best checkpoint from path:  /home/parth/Desktop/SEM6/RnD/Learning-From-Rules/data/TREC/outputs/learn2reweight_0.1_0.01_1/checkpoints/f_d_U/best.ckpt-498
Run mode is test_f
Running test_f
/home/parth/.local/lib/python3.8/site-packages/sklearn/metrics/classification.py:1436: UndefinedMetricWarning: Precision and F-score are ill-defined and being set to 0.0 in labels with no predicted samples.
  precision = _prf_divide(tp_sum, pred_sum,
test_f: precision:  [0.000 0.648 0.651 0.961 0.565 0.747]
test_f: recall:  [0.000 0.935 0.436 0.754 0.642 0.628]
test_f: f1_score:  [0.000 0.766 0.522 0.845 0.601 0.683]
test_f: support:  [  9 138  94  65  81 113]
test_f: accuracy:  0.684
test_f: avg_f1_score:  0.5694245986514627
test_f: classifier_loss:  1.1549703
